src/test/java/sk/stuba/fei/uim/vsa/pr1/StudentTest.java

```diff
@@ -140,7 +140,7 @@ void ST04_shouldUpdateStudent() {
             student = thesisService.updateStudent(student);
             assertNotNull(student);
             assertEquals(randomSetStringForTesting, getFieldValue(student, stringField, String.class));
-            student = thesisService.getStudent(Student01.aisId);
+            student = thesisService.getStudent(Student01.id);
```

Comment:
`Student01` je trieda v súbore [TestData](src/test/java/sk/stuba/fei/uim/vsa/pr1/utils/TestData), ktorá obsahuje iba súbor testovacích hodnôt, nemá nič s tím ako je implementovaná entita a už vôbec nemá atribút id.
Spúšťal si si to s tvojimi zmenami?

Reply:
Viem ale nevedel som ako to opraviť tak som to aspoň ilustroval aby ste vedeli o chybe.